lhotse/recipes/gigaspeech.py

```diff
@@ -0,0 +1,14 @@
+from typing import Optional, Sequence, Union
+from lhotse.utils import Pathlike, is_module_available
+
+
+def download_gigaspeech(
+        target_dir: Pathlike = '.',
+        dataset_parts: Optional[Union[str, Sequence[str]]] = "XS",
+        force_download: Optional[bool] = False
+) -> None:
+    if is_module_available('speechcolab'):
+        from speechcolab.datasets import gigaspeech
```

Comment:
I think the error message will be quite cryptic in this case (undefined symbol `gigaspeech`) -- how about:

```python
if not is_module_available('speechcolab'):
    raise ImportError('To download GigaSpeech, please install optional dependency: pip install speechcolab')
```